### ISSUE TITLE
[Project] 데이터베이스에 데이터를 만드는 방법

### ISSUE THREAD

**Issue by ByungHeeCha** [swsnu/swpp2019-team10#96]
저희 팀이 만든 recommendation system의 성능을 테스트 해보기 위해서 데이터를 만들어서 테스트 해보기로 했습니다. 그러나 직접 우리가 만드는 방법은 시간도 오래걸리고 성능 테스트에 필요없는 필드 (예를 들면, Review의 content는 recommendation에는 필요가 없잖아요?)가 많을것 같습니다. 혹시 빠른시간에 많은 데이터를 DB에 집어넣을 수 있는 방법이 없을끼요?

**Comment by ktaebum** [comment]
앗 안녕하세요
그런 부분에서는 사실상 어디선가 리뷰 데이터들을 다 크롤링 해와서 필요한 정보들만 넣는 게 가장 많이 집어넣을 수 있지 않나.. 하는 생각입니다.

**Comment by ByungHeeCha** [comment]
인터넷에 있는 리뷰데이터들은 우리의 리뷰 포맷과 달라서 어떻게 집어넣을지 모르겠네요... 예를들면 User model의 taste dictionary(user의 취향) 은 어떻게 넣을지 모르겠어요...

**Comment by ktaebum** [comment]
네 사실 외부에서 가져오는 데이터가 project에 딱 fit 되는 것을 기대하는 게 더 어렵습니다 😢 그래서 그 부분은 
데이터를 가져와서 -> 필요한 처리를 하고 -> 저장
을 구현을 하셔야할 것 같습니다

**Comment by ByungHeeCha** [comment]
그렇군요ㅠㅠ 감사합니다.
